.gitignore

```diff
@@ -91,3 +91,7 @@ ENV/
 
 # SQLite3
 db.sqlite3
+
+.DS_Store
+
+/migrations
```

Comment:
I think we should be pushing the migrations. Kindly remove it from this file.

Reply:
I was wondering whether or not the migrations should be included. Thanks for clarifying it.